test/src/e2e/changeParams.test.ts

```diff
@@ -429,7 +428,7 @@ describe("ChangeParams", function() {
         {
             await node.sdk.rpc.devel.stopSealing();
             const blockNumber = await node.sdk.rpc.chain.getBestBlockNumber();
-            const seq = await node.sdk.rpc.chain.getSeq(faucetAddress);
+            const seq = ( await node.rpc.chain.getSeq({address: faucetAddress.toString(), blockNumber:null}))!;
```

Comment:
DId you run the formatter?
It seems that the formatter will remove the white space of `( await`.